.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
     - TOX_ENV=py33-django16
     - TOX_ENV=py33-django17
     - TOX_ENV=py34-django16
-    - TOX_ENV=py34-django17
+    - TOX_ENV=py34-django17 DEPLOY_RELEASE=true
```

Comment:
I guess it will never be released, are we likely to allow to release a tag if the build fail?